twocaptchaapi/__init__.py

```diff
@@ -127,33 +129,75 @@ def get_load(self):
         """Obtains load statistics of the server."""
         return self.get(self.LOAD_URL, {}).text
 
+    def is_base64(self, string):
```

Comment:
I think we should rather put that into the module scope (right where the decorators live) and give it an underscore prefix (since it's rather internal stuff and doesn't depend on `self`).